### ISSUE TITLE
Volume-based weight in containers.

### ISSUE THREAD

**Issue by CUBE0R** [studio-minus/ppg-bugs-and-requests#1157]
**Feature description**
This would basically be the amount of liquid in a contain would add to its normal weight, based on the amount of liters in the container.

**Feature purpose**
This would allow for alot of interesting contraptions, and would be more realistic, you could hook this up to a scale and detector to make liter-measuring scales, or basically customized weight for exactly whatever you need, or even pump/remove weight to make a submarine submerge/float, and could even be used for air flight if you shift weight.

**Additional context**

**Comment by DindinYT37** [comment]
I agree but what's that on the picture?

**Comment by CUBE0R** [comment]
> I agree but what's that on the picture?

What do you mean?

**Comment by Drogo-man** [comment]
> I agree but what's that on the picture?

Search up "scale" on the workshop, and it should be the first result.

**Comment by CUBE0R** [comment]
> > I agree but what's that on the picture?
> 
> Search up "scale" on the workshop, and it should be the first result.

exactly, i just picked some scale on the workshop for demonstration, its right before the image in text.

**Comment by DindinYT37** [comment]
> > > I agree but what's that on the picture?
> > 
> > 
> > Search up "scale" on the workshop, and it should be the first result.
> 
> exactly, i just picked some scale on the workshop for demonstration, its right before the image in text.

That's an edit but okay anyway.

**Comment by CUBE0R** [comment]
> > > > I agree but what's that on the picture?
> > > 
> > > 
> > > Search up "scale" on the workshop, and it should be the first result.
> > 
> > 
> > exactly, i just picked some scale on the workshop for demonstration, its right before the image in text.
> 
> That's an edit but okay anyway
that edit was before you ever posted that comment though...
